app/component/front-page/FavouritesTabLabelContainer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Relay from 'react-relay';
+import connectToStores from 'fluxible-addons-react/connectToStores';
+import mapProps from 'recompose/mapProps';
+import some from 'lodash/some';
+import flatten from 'lodash/flatten';
+import RoutesRoute from '../../routes/RoutesRoute';
+import TabLabel from './FavouritesTabLabel';
+
+const hasDisruption = (routes) =>
+  some(flatten(routes.map(route => route.alerts.length > 0)));
+
+const alertReducer = mapProps(({ routes, classes, ...rest }) => ({
+  hasDisruption: hasDisruption(routes),
+  classes,
+  ...rest,
+}));
+
+const FavouritesTabLabel = Relay.createContainer(alertReducer(TabLabel), {
```

Comment:
This should probably be renamed to something else in order to avoid confusion with `./FavouritesTabLabel`